tests/utils/test_helpers.py

```diff
@@ -989,6 +989,40 @@ def test_get_pretty_types_names():
         assert pretty_result == expected
 
 
+@pytest.mark.parametrize(
+    "value, expected, exception",
```

Comment:
I like more the name `expectation` - as it's not necessarily an exception:
```suggestion
    "value, expected, expectation",
```

Reply:
Understand the comment but No exception is still an exception :)